services/salesforce.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const request = require('request-promise-native');
+const { SALESFORCE_AUTH } = require('../modules/secrets');
+
+class Salesforce {
```

Comment:
The service returns an instance of this class when calling `authorise`. Adds a little wrapper to handle passing the oath token along with the  request. We create a new connection each time we submit a form.

Reply:
Ooh our first `class`. Nice!